Subject: lib: on OpenVMS, include float+.h as float_plus.h

VSI C does not find a header whose name contains "+" (%CC-F-NOINCLFILEF
for "float+.h").  tools/prepare.sh copies lib/float+.h to
lib/float_plus.h; include it by that name on VMS.

--- a/lib/isnan.c	2026-10-04 11:47:13.540656011 +0100
+++ b/lib/isnan.c	2026-10-04 11:47:13.542414932 +0100
@@ -33,7 +33,11 @@
 #include <float.h>
 #include <string.h>
 
-#include "float+.h"
+#ifdef __VMS
+# include "float_plus.h"
+#else
+# include "float+.h"
+#endif
 
 #ifdef USE_LONG_DOUBLE
 # define FUNC rpl_isnanl
--- a/lib/signbitd.c	2026-10-04 11:47:13.547036708 +0100
+++ b/lib/signbitd.c	2026-10-04 11:47:13.547716490 +0100
@@ -21,7 +21,11 @@
 
 #include <string.h>
 #include "isnand-nolibm.h"
-#include "float+.h"
+#ifdef __VMS
+# include "float_plus.h"
+#else
+# include "float+.h"
+#endif
 
 #ifdef gl_signbitd_OPTIMIZED_MACRO
 # undef gl_signbitd
--- a/lib/signbitf.c	2026-10-04 11:47:13.551642230 +0100
+++ b/lib/signbitf.c	2026-10-04 11:47:13.552318948 +0100
@@ -21,7 +21,11 @@
 
 #include <string.h>
 #include "isnanf-nolibm.h"
-#include "float+.h"
+#ifdef __VMS
+# include "float_plus.h"
+#else
+# include "float+.h"
+#endif
 
 #ifdef gl_signbitf_OPTIMIZED_MACRO
 # undef gl_signbitf
--- a/lib/signbitl.c	2026-10-04 11:47:13.556291661 +0100
+++ b/lib/signbitl.c	2026-10-04 11:47:13.557278607 +0100
@@ -21,7 +21,11 @@
 
 #include <string.h>
 #include "isnanl-nolibm.h"
-#include "float+.h"
+#ifdef __VMS
+# include "float_plus.h"
+#else
+# include "float+.h"
+#endif
 
 #ifdef gl_signbitl_OPTIMIZED_MACRO
 # undef gl_signbitl
--- a/lib/vasnprintf.c	2026-10-04 11:47:13.561656274 +0100
+++ b/lib/vasnprintf.c	2026-10-04 11:47:13.565696873 +0100
@@ -107,7 +107,11 @@
 
 #if NEED_PRINTF_DOUBLE || NEED_PRINTF_LONG_DOUBLE || (NEED_WPRINTF_DIRECTIVE_LA && WIDE_CHAR_VERSION)
 # include <math.h>
-# include "float+.h"
+# ifdef __VMS
+#  include "float_plus.h"
+# else
+#  include "float+.h"
+# endif
 #endif
 
 #if NEED_PRINTF_DOUBLE || NEED_PRINTF_INFINITE_DOUBLE
